db/models.py

```diff
@@ -45,3 +55,69 @@ class MovieSession(models.Model):
 
     def __str__(self):
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models. DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self):
+        return f"{self.created_at}"
+
+
+class UniqueConstraints:
```

Comment:
extra class?

Reply:
It was automatically created by IDE when I made a grammar mistake. I even haven't seen it. Fixed.